modules/rosa-sts-ocm-role-creation.adoc

```diff
@@ -30,7 +31,9 @@ $ rosa create ocm-role
 $ rosa create ocm-role --admin
 ----
 +
-This command allows you create the role by specifying specific attributes. The following example output shows the "auto mode" selected, which lets the ROSA CLI (`rosa`) create your Operator roles and policies. See "Methods of account-wide role creation" in the Additional resources for more information.
+This command allows you create the role by specifying specific attributes. The following example output shows the "auto mode" selected, which lets the ROSA CLI (`rosa`) create your Operator roles and policies.
+See "Methods of account-wide role creation" in the Additional resources for more information.
```

Comment:
```suggestion
See "Methods of account-wide role creation" for more information.
```

https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#lead-in-link-sentence-style-for-non-xref-situations